<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ASHKOL" failures="2" tests="10" name="Test" time="28.427" errors="0" timestamp="2025-08-23T15:08:19 IST">
  <testcase classname="api.tests.UserTests" name="createUser" time="3.885"/>
  <testcase classname="api.tests.UserTests" name="getUser" time="7.686">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.UserTests.getUser(UserTests.java:59)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- getUser -->
  <testcase classname="api.tests.UserTests" name="updateUser" time="3.138"/>
  <testcase classname="api.tests.UserTests" name="deleteUser" time="1.622">
    <failure type="java.lang.AssertionError" message="expected [200] but found [404]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at api.tests.UserTests.deleteUser(UserTests.java:89)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- deleteUser -->
  <testcase classname="api.tests.DdUserTests" name="creatUsers" time="1.561"/>
  <testcase classname="api.tests.DdUserTests" name="creatUsers" time="1.587"/>
  <testcase classname="api.tests.DdUserTests" name="creatUsers" time="1.614"/>
  <testcase classname="api.tests.DdUserTests" name="getUsers" time="1.522"/>
  <testcase classname="api.tests.DdUserTests" name="getUsers" time="1.579"/>
  <testcase classname="api.tests.DdUserTests" name="getUsers" time="1.618"/>
</testsuite> <!-- Test -->
